checkov/sca_package_2/scanner.py

```diff
@@ -24,10 +28,35 @@ def __init__(self, pbar: ProgressBar | None = None, root_folder: str | Path | No
 
     def scan(self) -> Sequence[dict[str, Any]]:
         """run SCA package scan and poll scan results"""
-        pass
+        self.run_scan()
+        return []
 
     def run_scan(self) -> dict[str, Any]:
-        pass
+        logging.info("Start to scan package files.")
+
+        request_body = {
+            "branch": "",
+            "commit": "",
+            "path": os.path.join(bc_integration.repo_path, '') if bc_integration.repo_path else "",
```

Comment:
why `os.path.join(bc_integration.repo_path, '')` and not just `bc_integration.repo_path`?

Reply:
because `bc_integration.repo_path` is without a slash in the end and if the slash not exists `os.path.join(bc_integration.repo_path, '')` will add it

Reply:
ok, got it